v0/destinations/monetate/data/MONETATEScreen.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "destKey": "events",
+        "sourceKeys": "context.screen",
```

Comment:
this is the screen location in mobile sdk payload?

Reply:
Yes